src/screens/CreatePost/index.js

```diff
@@ -876,11 +868,10 @@ const CreatePost = () => {
             backRef={sheetBackRef}
             goBack={() => navigation.goBack()}
             continueToEdit={() => sheetBackRef.current.close()}
-            isAnonimity={typeUser}
           />
         </View>
       </ScrollView>
-      <Loading visible={loading} color={useDynamicColors(typeUser).primary} />
+      <Loading visible={loading} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color prop has been removed from the Loading component. If the loading indicator relied on dynamic coloring, make sure it's handled differently now.
```diff
- color={useDynamicColors(typeUser).primary}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->